crates/lib3h_protocol/src/protocol.rs

```diff
@@ -292,8 +294,10 @@ impl From<Lib3hToClient> for Lib3hServerProtocol {
             Lib3hToClient::Connected(connected_data) => {
                 Lib3hServerProtocol::Connected(connected_data)
             }
-            Lib3hToClient::Disconnected(disconnected_data) => {
-                Lib3hServerProtocol::Disconnected(disconnected_data)
+            Lib3hToClient::Unbound(_unbound_data) => {
+                Lib3hServerProtocol::Disconnected(DisconnectedData {
+                    network_id: "".into(),
```

Comment:
Don't quite understand this conversion.

Reply:
It's a good question.  Currently Disconnected is not being used by anything or in core, but it's the place this message was originally connected to in n3h (I think) so I reconnected it.  It should go away.